---
float64 temperatura
float64 umidita
float64 radiazione
